bin/build.sh

```diff
@@ -21,8 +21,14 @@ convert_to_pdf() {
     pdf_path="${file%.md}.pdf"
     html_path="${file%.md}.html"
 
-    # Use first header as the document title; remove all # characters and whitespace from the beginning
-    document_title=$(head -n 1 "$file" | sed -E "s/^#+\s*//")
+    if [[ "$file" =~ build/documents/(motions|minutes)/(([a-zA-Z0-9.\-]|[[:blank:]])+)\.md ]]; then
```

Comment:
Should the [] include an underscore (_) character too?

Reply:
Also, try [[:space:]] instead of blank. This stuff is explained [here](https://fabianlee.org/2020/01/29/bash-using-bash_rematch-to-pull-capture-groups-from-a-regex/).

Reply:
Filenames in motions and minutes don't include the "_" char, but I can add it if you think it'd be better.

I used [[:blank:]] instead of [[:space:]] because space includes many more chars that wouldn't be included in any filename anyway (afaik).

![Captura de pantalla 2024-09-01 a la(s) 12 50 03](https://github.com/user-attachments/assets/06d7e89b-9ece-48a6-a539-e26f47e4a6fa)

Reply:
Oh, my bad, I thought blank would be the more generic one. And yeah, I think it's best to include _ to future-proof it.